### ISSUE TITLE
Issue with wget.exe on the eval-win81x86-enterprise template

### ISSUE THREAD

**Issue by arizvisa** [boxcutter/windows#237]
When trying to use `wget.exe` to download files on the eval-win81x86-enterprise template, the version of `wget.exe` on https://eternallybored.org/misc/wget/current/wget.exe seems to not be compatible which results in evertying dependent on it failing.

Relevant logs from a failed build:

```
    vmware-iso: ==> Downloading "https://raw.githubusercontent.com/mitchellh/vagrant/master/keys/vagrant.pub" to "C:\Users\root\AppData\Local\Temp\vagrant\vagrant.pub"                                                                                                                                                         
==> vmware-iso: This version of C:\Windows\wget.exe is not compatible with the version of Windows you're running. Check your computer's system information and then contact the software publisher.                                                                                                                             
    vmware-iso: ==> Creating "C:\Users\root\.ssh"                                                                                                               
    vmware-iso: ==> Adding "C:\Users\root\AppData\Local\Temp\vagrant\vagrant.pub" to "C:\Users\root\.ssh\authorized_keys"                                       
    vmware-iso: ==> Disabling account password expiration for user "root"                                                                                       
    vmware-iso: Updating property(s) of '\\WINDOWS-R8N95SH\ROOT\CIMV2:Win32_UserAccount.Domain="WINDOWS-R8N95SH",Name="root"'                                   
    vmware-iso: Property(s) update successful.                                                                                                                  
```

**Comment by arizvisa** [comment]
Okay. So here's an issue.

The minimium supported ssl-based protocol for http://eternallybored.org is now Tls 1.2. So, that means older versions of Windows can't download `wget.exe` from it... Also, the latest version of `wget.exe` from that persons domain does not seem to work on all of the Windows platforms I've tried except for Server 2016 (probably Win10 as well). It's also ginormous...

I'm really considering allowing users to specify their own version of wget.exe to embed and giving them a default compile of one. Ideally due to PR #239 `wget.exe` should only be needed for versions of Windows that we're unable to use Powershell to download and so it's more of a fallback.

We can't put eternallybored.org's version of wget on a floppy because it's 4mb. This is crazy to me because I compile wget w/ msys2 and it's only 100k.

**Comment by arizvisa** [comment]
This was closed by PR #240.
